### ISSUE TITLE
Importing screenshot with space in the name are not url decoded?

### ISSUE THREAD

**Issue by palaniraja** [fastlane/fastlane#1448]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [ ] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description
I was initializing fastlane on an existing project. It downloaded all the meta and screenshots fine as I would expect. However when I try to run the *deliver* to push it to appstore, I found some the screenshots were missing in the generated *Preview.html* file. 

Then I renamed my screenshots %20 with _Space_ and it seems to work.

###Before
![Before - missing screenshot expect the first one](https://cloud.githubusercontent.com/assets/124864/13442196/db61329a-dff2-11e5-8669-1be7e61a3629.png)

###After
![After - Renaming %20 with space](https://cloud.githubusercontent.com/assets/124864/13442230/037f6d0a-dff3-11e5-8e9a-ccf25da23a18.png)

##### Complete output when running fastlane, including the stack trace and command used

```
deliver
[21:15:14]: Login to iTunes Connect (xxxx@xxxxxxxxx.com)
[21:15:18]: Login successful

+--------------------+-------------------------------------+
|                  deliver 1.10.3 Summary                  |
+--------------------+-------------------------------------+
| username           | xxxx@xxxxxxxxx.com              |
| app_identifier     | com.xxxxx.xxxxx |
| ipa                | CN.ipa                              |
| metadata_path      | ./fastlane/metadata                 |
| screenshots_path   | ./fastlane/screenshots              |
| skip_binary_upload | false                               |
| skip_screenshots   | false                               |
| app_version        | 2.2.2                               |
| skip_metadata      | false                               |
| force              | false                               |
| submit_for_review  | false                               |
| automatic_release  | false                               |
+--------------------+-------------------------------------+

[21:15:19]: Making sure the latest version on iTunes Connect matches '2.2.2' from the ipa file...
[21:15:20]: '2.2.2' is the latest version on iTunes Connect
[21:15:20]: Loading './fastlane/metadata/en-US/description.txt'...
[21:15:20]: Loading './fastlane/metadata/en-US/keywords.txt'...
[21:15:20]: Loading './fastlane/metadata/en-US/release_notes.txt'...
[21:15:20]: Loading './fastlane/metadata/en-US/support_url.txt'...
[21:15:20]: Loading './fastlane/metadata/en-US/marketing_url.txt'...
[21:15:20]: Loading './fastlane/metadata/en-US/name.txt'...
[21:15:20]: Loading './fastlane/metadata/en-US/privacy_url.txt'...
[21:15:20]: Loading './fastlane/metadata/copyright.txt'...
[21:15:20]: Loading './fastlane/metadata/primary_category.txt'...
[21:15:20]: Loading './fastlane/metadata/secondary_category.txt'...
[21:15:20]: Loading './fastlane/metadata/primary_first_sub_category.txt'...
[21:15:20]: Loading './fastlane/metadata/primary_second_sub_category.txt'...
[21:15:20]: Loading './fastlane/metadata/secondary_first_sub_category.txt'...
[21:15:20]: Loading './fastlane/metadata/secondary_second_sub_category.txt'...
[21:15:20]: Verifying the upload via the HTML file can be disabled by either adding
[21:15:20]: `force true` to your Deliverfile or using `deliver --force`
Does the Preview on path './Preview.html' look okay for you? (y/n)
n

[!] Did not upload the metadata, because the HTML file was rejected by the user
```

### Configuration Files

Please copy the complete content of your `Fastfile` and any other configuration files you use below:

Fastfile:
```ruby
# INSERT CONTENT HERE
```

### Environment

fastlane version (fastlane -v): fastlane 1.63.1
Do you use bundler, rbenv or rvm? NO

**Comment by vpolouchkine** [comment]
@palaniraja Is this still an issue using latest versions?

**Comment by palaniraja** [comment]
Yes. I still see the %20 in latest (fastlane 1.81.0) too.

![screen shot 2016-04-09 at 8 07 42 pm](https://cloud.githubusercontent.com/assets/124864/14405820/04c22846-fe8f-11e5-94cf-cb9ad9a3f18a.png)

**Comment by TKBurner** [comment]
@palaniraja Thanks for following up. We are going to look this over as a team. Hope that you have a great day! 👍

**Comment by mfurtak** [comment]
I have reproduced this problem locally for file names that contain `%20`

**Comment by mfurtak** [comment]
I've also verified that `deliver` is not responsible adding `%20` to the file names.

**Comment by mfurtak** [comment]
Hi @palaniraja - we've merged a fix for this in #4196, which will be in the next release! :rocket:

**Comment by mfurtak** [comment]
Hi @palaniraja - this shipped as part of [`deliver` 1.11.1](https://github.com/fastlane/fastlane/releases/tag/deliver%2F1.11.1)! 🚚 ✨

**Comment by palaniraja** [comment]
Thanks for the heads up. Sorry I didn't get a chance to check this week. I'll update if I see any issue.

Thank you all for the fix.
